### DISCUSSION TITLE
implementing geom parameters which are functions

### DISCUSSION THREAD

**Discussion by dannyparsons** [IDEMSInternational/R-Instat#9786] [General]
We currently only implement geom parameters which are single values. Some can take functions e.g.
`geom_point(pch = 21, position = position_jitterdodge())`

using:
```
position_jitterdodge(jitter.width = NULL, jitter.height = 0,
  dodge.width = 0.75)
```
This could be implemented as separate single value parameters e.g.
jitterdodge
jitterdodge:jitter.width
jitterdodge:jitter.height
jitterdodge:dodge.width
etc.
where these controls are 'linked' in some way.

**Comment by Ogik99** [comment]
@dannyparsons  should we go ahead with the implementation of this

**Comment by dannyparsons** [comment]
We'll need to have the full system in place for linking parameters in this way and decide how to implement. So it needs to be conceptualised first but happy if someone wants to start putting thought into this.
